drafts/draft-ietf-ippm-ioam-data.xml

```diff
@@ -328,10 +328,10 @@
       operational and telemetry information in the packet while the packet
       traverses a path between two points in the network. This document
       discusses the data fields and associated data types for in-situ OAM.
-      In-situ OAM data fields can be embedded into a variety of transports
-      such as NSH, Segment Routing, Geneve, native IPv6 (via extension
-      header), or IPv4. In-situ OAM can be used to complement OAM mechanisms
-      based on e.g. ICMP or other types of probe packets.</t>
+      In-situ OAM data fields can be embedded into a variety of protocols such
```

Comment:
s/embedded/encapsulated/

We use "encapsulated" at the term for embedding IOAM into protocols throughout the IOAM specs